pyproject.toml

```
@@ -18,10 +18,8 @@ dependencies = ["numpy"]
 
 [project.optional-dependencies]
 test = ["pytest", "scipy", "celerite"]
-pymc3 = ["pymc3>=3.9", "numpy<1.22", "xarray<2023.10.0"]
-theano = ["pymc3>=3.9", "numpy<1.22", "xarray<2023.10.0"]
-pymc = ["pymc>=5.9.2"]
-jax = ["jax"]
+pymc = ["pymc>=5.26.1"]
+jax = ["jax>=0.8.0,<0.9.0", "jaxlib>=0.8.0,<0.9.0"]
```

A jaxlib pin shouldn't be needed! And I'd recommend just doing `>=0.8.0` without the upper limit.

```suggestion
jax = ["jax>=0.8.0"]
```

---

Ok I disagree but it's your call!

---

By the way, the last JAX version compatible with celerite2 was 0.4.x; from JAX versions 0.5-0.8 it has been incompatible.